Final Project/Portfolio/contact.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Ashley Sullivan | Front End Developer &amp; Designer</title>
+	<meta name="description" content="">
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link href='https://fonts.googleapis.com/css?family=Raleway:300,400,600' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+	<script src="js/main.js"></script>
+</head>
+<body>
+	<header class="contact-header">
+		<div class="content">
+			<h1 class="contact-h1"><a href="index.html">Ashley Sullivan</a></h1>
+		</div>
+	</header>
+	<div class="contact content">
+		<h2>Contact</h2>
+		<p>I love meeting new people and would love to meet you. If you&rsquo;d like to discuss code, design, freelance opportunities or just to chat, please fill out the form below and I&rsquo;ll respond when I can.</p>
+		<p class="required">I tried to keep this form short and sweet, so all fields are required.</p>
+		<form method="post" action="https://formspree.io/ashley@ashleysullivan.co" enctype="text/plain">
+			<label for="Name">Name:</label>
+			<input type="text" name="Name" value="Your Name" placeholder="Your Name" id="Name"></input>
+			<label for="Email">Email:</label>
+			<input type="text" name="Email" value="Email Address" placeholder="Email Address" id="Email"></input>
+			<label for="Subject">Subject:</label>
+			<input type="text" name="Subject" value="What do you want to talk about?" placeholder="What do you want to talk about?" id="Subject"></input>
+			<label for="Message">Message:</label>
+			<textarea name="Message" cols="30" rows="10" placeholder="Tell me what you're thinking." id="Message">Tell me what you're thinking.</textarea>
+			<input type="submit">
+			<input type="reset">
+		</form>
+	</div>
+	<footer>
+		<div class="content">
+			<p><script type="text/javascript">copyrightDate();</script>Ashley Sullivan - All Rights Reserved.</p>
```

Comment:
You *might* want to take a look at running this function in JS and changing the HTML of this element opposed to running JS inline on the page. Just to separate the concerns of the technologies.